docs/resources/tanzu_kubernetes_cluster.md

```diff
@@ -318,6 +322,13 @@ resource "tanzu-mission-control_tanzu_kubernetes_cluster" "tkgs_cluster" {
 }
 ```
 
+## Import TKG Cluster
+The resource ID for importing an existing TKG cluster should be comprised of a full cluster name separated by '/'.
+
```

Comment:
Usage of TKG cluster, seems generic and incorrect. To my recollection its TKG 2.x cluster-class based cluster and you can't say import TKG 1.x cluster using said steps.

Reply:
fixed.